### PR TITLE
[EPMRPP-37466] Dao unit tests

### PR DESCRIPTION
**Pull request by ihar-kahadouski**

### CODE REVIEW

File: src/test/java/com/epam/ta/reportportal/entity/enums/LaunchModeEnumTest.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+ * @author <a href="mailto:ihar_kahadouski@epam.com">Ihar Kahadouski</a>
+ */
+public class LaunchModeEnumTest {
+
+	private Map<LaunchModeEnum, List<String>> allowed;
+	private List<String> disallowed;
+
+	@Before
+	public void setUp() throws Exception {
+		allowed = Arrays.stream(LaunchModeEnum.values()).collect(Collectors.toMap(it -> it, it -> permute(it.name())));
```
Comment (pbortnik) [inline src/test/java/com/epam/ta/reportportal/entity/enums/LaunchModeEnumTest.java]: same as with other enums

File: src/test/java/com/epam/ta/reportportal/entity/enums/LogLevelTest.java [hunk 2]
```diff
@@ -1,65 +1,116 @@
...
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static com.epam.ta.reportportal.entity.enums.EnumTestHelper.permute;
+import static org.junit.Assert.*;
 
 /**
- * @author <a href="mailto:ivan_budayeu@epam.com">Ivan Budayeu</a>
+ * @author <a href="mailto:ihar_kahadouski@epam.com">Ihar Kahadouski</a>
```
Comment (pbortnik) [inline src/test/java/com/epam/ta/reportportal/entity/enums/LogLevelTest.java]: discuss with Ivan about these tests

File: src/test/java/com/epam/ta/reportportal/entity/enums/converter/AttributeConverterTest.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+		allowedValues.forEach((key, value) -> value.forEach(it -> assertEquals(key, converter.convertToEntityAttribute(it))));
+	}
+
+	@Test
+	public void convertToEntityAttributeFail() {
+		thrown.expect(ReportPortalException.class);
+		converter.convertToEntityAttribute("wrong parameter");
+	}
+
+	protected abstract void doTest();
```
Comment (pbortnik) [inline src/test/java/com/epam/ta/reportportal/entity/enums/converter/AttributeConverterTest.java]: let's name this method more clear to understand what it should do

File: src/test/java/com/epam/ta/reportportal/entity/project/ProjectUtilsTest.java [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+		final Map<String, String> configParameters = ProjectUtils.getConfigParameters(getProjectAttributes());
+		assertNotNull(configParameters);
+		assertTrue(!configParameters.isEmpty());
+		assertThat(configParameters).isEqualTo(getAttributeWithValuesMap().entrySet()
+				.stream()
+				.collect(Collectors.toMap(it -> it.getKey().getName(), Map.Entry::getValue)));
+	}
+
+	@Test
+	public void getOwner() {
```
Comment (pbortnik) [inline src/test/java/com/epam/ta/reportportal/entity/project/ProjectUtilsTest.java]: ?

Comment (ihar-kahadouski) [reply src/test/java/com/epam/ta/reportportal/entity/project/ProjectUtilsTest.java]: private static final String OWNER = "OWNER";

Comment (pbortnik) [reply src/test/java/com/epam/ta/reportportal/entity/project/ProjectUtilsTest.java]: it is empty here

**Pull request merged**